pandas/io/parsers/base_parser.py

```diff
@@ -798,6 +799,26 @@ def _do_date_conversions(self, names, data):
 
         return names, data
 
+    def _check_data_length(self, columns: List[str], data: List[ArrayLike]):
+        """Checks if length of data is equal to length of column names. One set of
+        trailing commas is allowed.
```

Comment:
Short summary should be one line. Second sentence can be moved to extended summary I think.

Reply:
Done